docs/cli-reference/ark_schedule_describe.md

```diff
@@ -0,0 +1,39 @@
+## ark schedule describe
+
+Describe schedules
+
+### Synopsis
+
+
+Describe schedules
+
+```
+ark schedule describe [flags]
+```
+
+### Options
+
+```
+  -h, --help                        help for describe
+      --label-columns stringArray   a comma-separated list of labels to be displayed as columns
+  -o, --output string               Output display format. For create commands, display the object but do not send it to the server. Valid formats are 'table', 'json', and 'yaml'. (default "table")
```

Comment:
For the describe commands, the `-o/--output`, `lable-columns`, and `--show-labels` flags are ignored. It looks like they're added via `cmd/util/output` for every command, regardless of whether they're used or not.

Functionally, not a big deal, but may be a UX nit.